### PR TITLE
Update Laravel dependencies to 5.7, replace Laravel helper calls

### PR DESCRIPTION
**Pull request by stancl**
Hi,

I changed the required versions of illiminate packages to 5.7 and replaced all global helper calls with respective Arr or Str calls, like https://github.com/laravel/framework/pull/26898. For future 5.9 support.

All tests pass so everything should be okay.

### CODE REVIEW

**Comment by stancl** [comment]
I also reordered `use` statements so they're now in line with the way Laravel does it.

**Comment by damiani** [comment]
The `Arr` and `Str` replacements look great, but for this project, we prefer to alphabetize `use` statements, unlike the Laravel core. (There's a PHP CS Fixer config in the repo, `.php_cs`, which I need to document in the readme.) If you can revert the two commits that relate to re-ordering `use` statements, I can merge this in. Thanks!

**Comment by stancl** [comment]
Okay, reverted. Though I cannot guarantee that `use` statements introduced by me follow the alphabetic order. I'll see if I can perhaps run PHP CS Fixer on all files?

File: src/Collection/Collection.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 use Closure;
 use Illuminate\Support\Collection as BaseCollection;
+use Illuminate\Support\Arr;
```
Comment (stancl) [inline src/Collection/Collection.php]: My `use` statements were simply appended to the end.

**Comment by damiani** [comment]
Looks great, I'll run fixer after merge. Thanks for your contribution!

**Pull request merged**